nipype/algorithms/mesh.py

```diff
@@ -22,9 +22,40 @@
 from ..interfaces.base import (BaseInterface, traits, TraitedSpec, File,
                                BaseInterfaceInputSpec)
 from warnings import warn
+
+
+have_tvtk = False
+
+try:
+    from tvtk.api import tvtk
```

Comment:
the problem is that this import results in a sys exit call, if tvtk exists and DISPLAY is unavailable. i couldn't figure out a way to trap it. in that setting it doesn't generate an ImportError. it just plain kills the interpreter.